charts/kubewarden-controller/Chart.yaml

```diff
@@ -38,7 +38,7 @@ annotations:
   catalog.cattle.io/requests-cpu: "250m"
   catalog.cattle.io/requests-memory: "50Mi"
   catalog.cattle.io/rancher-version: ">= 2.6.0-0 <= 2.7.100-0" # Chart will only be available for users in the specified Rancher version(s), here its 2.5.0-2.5.99. This _must_ use build metadata or it won't work correctly for future RC's.
-  catalog.cattle.io/upstream-version: "1.3.1" # The version of the upstream chart or app. It prevents the unexpected "downgrade" when upgrading an installed chart that uses our 100.x.x+upVersion version schema.
+  catalog.cattle.io/upstream-version: "1.4.0" # The version of the upstream chart or app. It prevents the unexpected "downgrade" when upgrading an installed chart that uses our 100.x.x+upVersion version schema.
```

Comment:
Is this right? I think it is. But I would like to double check. The `catalog.cattle.io/upstream-version` should contains the chart version, right?

Reply:
yes, I think this is right. CC @jordojordo , @viccuad

Reply:
Yes that's correct.

Reply:
I'm pretty sure that our expectations are right: 
https://github.com/kubewarden/helm-charts/commit/84e14351b0014348c741ca8e740db94aab5326c7
https://github.com/kubewarden/helm-charts/commit/621a56ff6165db689c8a5f0b1564bb2c4d006088